manim/animation/animation.py

```diff
@@ -117,6 +118,10 @@ def update(self, alpha):
         This method shouldn't exist, but it's here to
         keep many old scenes from breaking
         """
+        logger.warning(
+             "animation.update() has been deprecated. "
+             "Please use animation.interpolate() instead."
```

Comment:
black complains about incorrect indentation, please fix:

```suggestion
            "animation.update() has been deprecated. "
            "Please use animation.interpolate() instead."
```

Otherwise, this looks good to me!

Reply:
Ah, you were faster! 👍

Reply:
I just saw it, too! Sent a commit, let's see if black will complain now.